mteb/tasks/Classification/multilingual/LanguageClassification.py

```diff
@@ -66,7 +66,58 @@ class LanguageClassification(AbsTaskClassification):
 }""",
         descriptive_stats={
             "n_samples": {"test": 2048},
-            "avg_character_length": {"test": 107.8},
+            "avg_character_length": {
+                "test": {
+                    "num_texts": 2048,
+                    "num_labels": 2048,
+                    "average_text_length": 109.546875,
+                    "num_label_17": 102,
+                    "num_label_0": 102,
```

Comment:
Not entirely sure what is going on here? A documentation on the field might help (see above)

Reply:
num_label_n is count how many times label occurred

Reply:
hmm but that will always be the same as num_texts, we should instead use "number of samples". 

I also think it might be worth writing some of these out (no reason to write it like variable names in string).

Reply:
No, this refers to how often each label appears.

Reply:
A so num_label_n is how many times label N occurred. It might be better as a nested dict. "label frequency" {"1": 102, ...}